src/Component.php

```diff
@@ -19,6 +19,10 @@ protected function run(): void
         try {
             /** @var Config $config */
             $config = $this->getConfig();
+            if ($config->getAllowSourceProjectId() && $config->getProjectId() !== $config->getAllowSourceProjectId()) {
```

Comment:
Tady mi to podle error message přijde, že by se ty ID nesmí rovnat a né musí? Ale spíš tomu jen nerozumím, protože to máš pokryté testem...

Reply:
však tam je ` !== ` tj že když se nerovnají - tj že když si v jiným než v povoleným projektu

Reply:
Ok, já jen pro jistotu.